wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'seed' => 'create',
+  'reseed' => 'recreate',
+  'load' => 'seed'
```

Comment:
The only real difference between seed and load is that seed -> rake wagn:create clears the tables first.